pygmt/tests/test_datasets_samples.py

```diff
@@ -173,3 +173,16 @@ def test_earth_relief_holes():
     npt.assert_allclose(grid.min(), -4929.5)
     # Test for the NaN values in the remote file
     assert grid[2, 21].isnull()
+
+
+def test_maunaloa_co2():
+    """
+    Check that the @MaunaLoa_CO2.txt dataset loads without errors.
+    """
+    data = load_sample_data(name="maunaloa_co2")
+    assert data.shape == (730, 2)
+    summary = data.describe()
+    assert summary.loc["min", "date"] == 1958.2027
+    assert summary.loc["max", "date"] == 2019.3699
```

Comment:
Is the `date` column a floating point column? Should we convert it to a proper pandas datetime dtype?

Reply:
It is a floating point number in the original dataset. My concern with converting to a date time object is that the GMT API won't recognize that input.

Reply:
PyGMT does work with datetime I'm sure. See https://www.pygmt.org/v0.6.1/tutorials/advanced/date_time_charts.html?highlight=datetime

Reply:
@GenericMappingTools/pygmt-maintainers Does anyone else have an input on this? My thought is to maintain the original date type in the remote file as a float, but I don't feel that strongly.

Reply:
I'm OK with a float-type date.